paasta_tools/paasta_maintenance.py

```diff
@@ -542,23 +567,6 @@ def paasta_maintenance():
     action = args.action
     hostnames = args.hostname
 
-    if action not in [
-            'down',
-            'drain',
-            'is_host_down',
-            'is_host_drained',
-            'is_host_draining',
-            'is_host_past_maintenance_end',
-            'is_host_past_maintenance_start',
-            'is_safe_to_kill',
-            'schedule',
-            'status',
-            'undrain',
-            'up',
-    ]:
-        print "action must be 'drain', 'undrain', 'down', 'up', 'status', or 'schedule'"
-        return
-
```

Comment:
The arg-parser already has "choices", which should be enough
https://docs.python.org/3/library/argparse.html#choices